app/views/root/_base.html.erb

```diff
@@ -30,11 +32,21 @@
   <% if @banner_notification.present? %>
     <%= @banner_notification %>
   <% end %>
+
+  <% if @emergency_banner %>
```

Comment:
@nickcolley Just the styling (or the lack of it on the default banner). If the emergency is ever restyled so that it looks identical on all pages, including the homepage, this `is_homepage` check can be removed.